Oracle/pcornet_loader.sql

```diff
@@ -75,6 +75,22 @@ begin
     and rx_frequency is null
     and rx_refills is null
   ;
+  
+   /* Remove ICD-9 rows from the procedures table that don't follow CDM specifications
+   ICD-9-CM (09): 3-4 numbers, including a decimal
+   */
+    delete 
+    from procedures
+    where PX_type = '09'
+    and  not regexp_like(px, '[.?0-9-]{3,4}'); 
```

Comment:
I was kinda thinking that this delete would be a one-time manual thing and that going forward, we would avoid inserting the bad ones in the first place.

But I guess this is OK.